reference/src/clientimpl/src/sa_crypto_cipher_process_last.c

```diff
@@ -74,28 +68,22 @@ sa_status sa_crypto_cipher_process_last(
                     break;
                 }
 
+                if (out->context.clear.offset > out->context.clear.length) {
```

Comment:
Are these some of the integer overflow tests from the other PR?

Reply:
Yes. Removed from here.